src/stories/avatar/InternalAvatar.tsx

```diff
@@ -0,0 +1,19 @@
+import styled from "styled-components";
+import LogoIcon from "../../assets/logo-icon.svg";
+
+const StyledInternalAvatar = styled.div`
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  cursor: pointer;
```

Comment:
questo lo toglierei sennò sembra che ci puoi cliccare

Reply:
in realtà ho notato che è di default così che si vede sempre il pointer ma comunque lo levo da qua intanto

Reply:
fatto